scripts/us_epa/ejscreen/config.json

```diff
@@ -0,0 +1,99 @@
+{
+  "YEARS": [ "2015", "2016", "2017", "2018", "2019", "2020", "2021", "2022", "2023", "2024"],
+  "NORM_CSV_COLUMNS": ["ID", "DSLPM", "CANCER", "RESP", "OZONE", "PM25"],
+  "NORM_CSV_COLUMNS1": ["ID", "DSLPM", "OZONE", "PM25"],
+  "CSV_COLUMNS_BY_YEAR": {
+    "2015": ["FIPS", "dpm", "cancer", "resp", "o3", "pm"],
+    "2016": ["ID", "DSLPM", "CANCER", "RESP", "OZONE", "PM25"],
+    "2017": ["ID", "DSLPM", "CANCER", "RESP", "OZONE", "PM25"],
+    "2018": ["ID", "DSLPM", "CANCER", "RESP", "OZONE", "PM25"],
+    "2019": ["ID", "DSLPM", "CANCER", "RESP", "OZONE", "PM25"],
+    "2020": ["ID", "DSLPM", "CANCER", "RESP", "OZONE", "PM25"],
+    "2021": ["ID", "DSLPM", "CANCER", "RESP", "OZONE", "PM25"],
+    "2022": ["ID", "DSLPM", "CANCER", "RESP", "OZONE", "PM25"],
+    "2023": ["ID", "DSLPM", "CANCER", "RESP", "OZONE", "PM25"],
+    "2024": ["ID", "DSLPM", "OZONE", "PM25"]
+  },
+  "ZIP_FILENAMES": {
+    "2015": "EJSCREEN_20150505.csv",
+    "2016": "EJSCREEN_V3_USPR_090216_CSV",
+    "2017": null,
+    "2018": "EJSCREEN_2018_USPR_csv",
+    "2019": "EJSCREEN_2019_USPR.csv",
+    "2020": "EJSCREEN_2020_USPR.csv",
+    "2021": "EJSCREEN_2021_USPR.csv",
+    "2022": "EJSCREEN_2022_with_AS_CNMI_GU_VI.csv",
+    "2023": "EJSCREEN_2023_BG_with_AS_CNMI_GU_VI.csv",
+    "2024": "EJScreen_2024_Tract_with_AS_CNMI_GU_VI.csv"
+  },
+  "FILENAMES": {
```

Comment:
What's the difference between ZIP_FILENAMES and FILENAMES. Can we retain just one if the other can be derived from it?